.gitignore

```diff
@@ -3,6 +3,7 @@
 *.gcno
```

Comment:
These additions are good.  However, we'd prefer to separate this out into a separate PR since they are unrelated to the bug being fixed.  Would also be good to add `output` to the list as well.